scripts/core/editor3/components/handlePastedText.ts

```diff
@@ -82,6 +82,10 @@ export function handlePastedText(text: string, _html: string): DraftHandleValue
         return 'handled';
     }
 
+    if (this.props.cleanPastedHtml) { // force plain text, but keep line breaks
```

Comment:
Shouldn't pasting multi-line text always work, not only when you have `cleanPastedHtml` enabled?

Reply:
it works, but it keeps some html if it's enabled like headlines, lists, probably all supported blocks.

Reply:
I've checked the PR locally and it doesn't work without enabling `cleanPastedHtml`

![lines](https://user-images.githubusercontent.com/2076953/68389331-fa8a1e80-0162-11ea-9f21-3326f9e279ce.gif)

Reply:
yep that's the idea, you have to enable that in content profile

Reply:
or you mean it should preserve line breaks when pasting text? that's probably how it works now on master too

Reply:
> yep that's the idea, you have to enable that in content profile

To enable what? preserving lines when pasting? It should work out the box. It does on draftjs demo website. I know it doesn't work on master either, but if you are working on the feature, it would be good to fix it in general instead of addressing a single use-case.

Reply:
sure, I thought you're talking about removing formatting